qa/L0_backend_python/response_sender/response_sender_test.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Side note: Can we target these test changes into `main` so there is less to review in eventual merge to main if it tests existing features? This way we can also see if you make any changes to these tests in future work.

Reply:
Sure. @Tabrizian are you also ok merging this directly into main?

Reply:
Let's keep it in the side branch. I don't think it is testing existing features. If it is merged to `main` it will fail since you can't use `response_sender` in non-decoupled mode.